content_store_controller.go

```diff
@@ -13,54 +14,69 @@ import (
 )
 
 type ContentStoreController struct {
-	arbiter      *urlarbiter.URLArbiter
-	contentStore *contentstore.ContentStoreClient
+	arbiter           *urlarbiter.URLArbiter
+	liveContentStore  *contentstore.ContentStoreClient
+	draftContentStore *contentstore.ContentStoreClient
 }
 
 type ContentStoreRequest struct {
 	PublishingApp string `json:"publishing_app"`
 }
 
-func NewContentStoreController(arbiterURL, contentStoreURL string) *ContentStoreController {
+func NewContentStoreController(arbiterURL, liveContentStoreURL, draftContentStoreURL string) *ContentStoreController {
 	return &ContentStoreController{
-		arbiter:      urlarbiter.NewURLArbiter(arbiterURL),
-		contentStore: contentstore.NewClient(contentStoreURL),
+		arbiter:           urlarbiter.NewURLArbiter(arbiterURL),
+		liveContentStore:  contentstore.NewClient(liveContentStoreURL),
+		draftContentStore: contentstore.NewClient(draftContentStoreURL),
 	}
 }
 
-func (cs *ContentStoreController) PutContentStoreRequest(w http.ResponseWriter, r *http.Request) {
-	urlParameters := mux.Vars(r)
+func (controller *ContentStoreController) PutDraftContentStoreRequest(w http.ResponseWriter, r *http.Request) {
+	controller.registerWithURLArbiterAndForward(w, r, func(path string, requestBody []byte) {
+		controller.doDraftContentStoreRequest("PUT", strings.Replace(path, "draft-", "", -1), requestBody, w)
+	})
+}
 
-	requestBody, err := ioutil.ReadAll(r.Body)
-	if err != nil {
-		renderer.JSON(w, http.StatusInternalServerError, err)
-		return
-	}
+func (controller *ContentStoreController) PutContentStoreRequest(w http.ResponseWriter, r *http.Request) {
```

Comment:
@jamiecobbett: i've created https://github.com/alphagov/publishing-api/pull/23 to address the naming.